scripts/west_commands/build.py

```diff
@@ -143,6 +143,13 @@ def do_add_parser(self, parser_adder):
                            Do not use this option with manually specified
                            -DSNIPPET... cmake arguments: the results are
                            undefined''')
+        group.add_argument('-i', '--shield', dest='shields',
```

Comment:
we don't need a short argument variant for all arguments.

And especially in this case I'm not fond of reserving `-i` for something like shield.
It could happen in future that a long arg starting with `i` is introduced and therefor making it natural to use `-i` for such argument.

Reply:
That's fair. I wasn't too happy with `-i` myself. I have removed it, leaving only the long option.